sa/sa_test.go

```diff
@@ -769,6 +770,7 @@ func TestAddIssuedNames(t *testing.T) {
 func TestDeactivateAuthorization(t *testing.T) {
 	sa, _, cleanUp := initSA(t)
 	defer cleanUp()
+	features.Set(map[string]bool{"AllowAccountDeactivation": true})
```

Comment:
Probably there should be something in the gating docs about it, I'd entirely forgotten about it until now ._.